app/reducers/dataFeeds.test.js

```diff
@@ -32,28 +36,90 @@ describe('dataFeeds', () => {
   describe('DATA_FEED_REMOVED_EVENT', () => {
 
     it('removes the data feed with the matching address', () => {
-      const actual = dataFeeds([mockAddr(0)], dataFeedRemovedAction(mockAddr(0)))
+      const actual = dataFeeds([mockDataFeed(0)], dataFeedRemovedAction(mockDataFeed(0)))
       const expected = []
       assert.deepEqual(actual, expected)
     })
 
     it('does not remove any data feeds if there is no matching address', () => {
-      const actual = dataFeeds([mockAddr(1)], dataFeedRemovedAction(mockAddr(0)))
-      const expected = [mockAddr(1)]
+      const actual = dataFeeds([mockDataFeed(1)], dataFeedRemovedAction(mockDataFeed(0)))
+      const expected = [mockDataFeed(1)]
       assert.deepEqual(actual, expected)
     })
 
   })
+
+  describe('DATA_FEED_LATEST_RESULT_LOADED', () => {
+    let currentResult, lastUpdated, address, nullLastUpdated
+
+    beforeEach(async () => {
+      nullLastUpdated = 0
+      lastUpdated = Date.now()
+      currentResult = uintToBytes32(123)
+      address = mockAddr(1)
+    })
+
+    it('adds formatted current result information', async () => {
+      const output = dataFeeds([mockDataFeed(0), mockDataFeed(1)], dataFeedLatestResultLoaded({ address, lastUpdated, currentResult }))
+      const dataFeed = _.find(output, dataFeedObj => { if (dataFeedObj.address == address) { return dataFeedObj } })
+      assert.deepEqual(dataFeed.currentResult, formatResult(currentResult, lastUpdated))
+    })
+
+    it('adds formatted lastUpdated information', async () => {
+      const output = dataFeeds([mockDataFeed(0), mockDataFeed(1)], dataFeedLatestResultLoaded({ address, lastUpdated, currentResult }))
+      const dataFeed = _.find(output, dataFeedObj => { if (dataFeedObj.address == address) { return dataFeedObj } })
```

Comment:
is `_.find(output, { address })` not working?

Reply:
fixed